app/modules/intelligence/agents/chat_agents/pydantic_agent.py

```
@@ -216,14 +189,26 @@ async def run_stream(
                                         ],
                                         citations=[],
                                     )
                                 if isinstance(event, FunctionToolResultEvent):
+                                    summary = ""
+                                    if (
+                                        isinstance(event.result.content, List)
+                                        and len(event.result.content) > 0
+                                    ):
+                                        summary = str(event.result.content[0])
+                                    elif isinstance(event.result.content, str):
+                                        summary = event.result.content
```

Ruff (0.8.2)

Local variable `summary` is assigned to but never used

Remove assignment to unused variable `summary`

(F841)